mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -190,6 +211,27 @@
       >
         Set to <Null />
       </ButtonMenuItem>
+      <!-- Column Attributes -->
+      <ColumnHeaderContextMenu {processedColumn} />
+      <!-- Column Attributes end -->
+      <!-- Row -->
+      <LinkMenuItem
+        icon={iconLinkToRecordPage}
+        href={$storeToGetRecordPageUrl({ recordId: rId }) || ''}
+      >
+        Go To Linked Record
+      </LinkMenuItem>
+      {#if canEditTableRecords}
+        <ButtonMenuItem
+          on:click={handleDeleteRecords}
+          danger
+          icon={iconDeleteMajor}
+        >
+          Delete Record
+        </ButtonMenuItem>
+      {/if}
```

Comment:
This block is present within `RowContextOptions`. We need to compose that component here instead of duplicating code.

Reply:
mathesar_service_dev   |     autoreload.check_errors(django.setup)()
mathesar_service_dev   |   File "/usr/local/lib/python3.9/site-packages/django/utils/autoreload.py", line 53, in wrapper
mathesar_service_dev   |     fn(*args, **kwargs)
mathesar_service_dev   |   File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
mathesar_service_dev   |     apps.populate(settings.INSTALLED_APPS)
mathesar_service_dev   |   File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 91, in populate
mathesar_service_dev   |     app_config = AppConfig.create(entry)
mathesar_service_dev   |   File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 116, in create
mathesar_service_dev   |     mod = import_module(mod_path)
mathesar_service_dev   |   File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
mathesar_service_dev   |     return _bootstrap._gcd_import(name[level:], package, level)
mathesar_service_dev   |   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
mathesar_service_dev   |   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
mathesar_service_dev   |   File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
mathesar_service_dev   | ModuleNotFoundError: No module named 'whitenoise'


I am getting all these errors now that I tried to re-run the branch locally, these were not present before and I haven't even touched these files on which the errors are being showed up.
Do you know any possible solution for this?

Reply:
I even tried removing all the things that I edited and 'pip installing whitenoise'. Still it shows the same error on this branch. Develop branch works fine.
What should I do in that case?

Reply:
> @TafreedAhmad
> 
> I've added a few comments.
> 
> Apart from them, I'd like to see dividers between the cell, column, and row options. There is a component called `MenuDivider` which can be used.

![image](https://user-images.githubusercontent.com/107949435/228393340-c36dbc6c-22a9-4f5c-a4bc-a6ccb8831a43.png)

Like this?

Reply:
Done!